src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -65,3 +66,53 @@
   height: 1px;
   overflow: hidden;
 }
+
+@include sky-theme-modern {
+  .sky-file-attachment-none {
+    font-size: 13px;
```

Comment:
I noticed the `13px` value is being used here, and in the label component (above). Is there a SCSS variable for that, or no?

Reply:
Yep, done.